sampleconfig/core.yaml

```diff
@@ -731,7 +731,7 @@ ledger:
 ###############################################################################
 operations:
     # host and port for the operations server
-    listenAddress: 127.0.0.1:9443
+    listenAddress: 127.0.0.1:9444
```

Comment:
There is a long history of peer operations port being 9443, so I wouldn't change it as production users may use it.

Typically orderer and peer run in their own container so there is no such conflict. Perhaps it would be better to update the dev mode instructions and mention to set the options listenAddress?

Reply:
I have done the changes in the peer dev mode instructions, please a look.